src/background/services/keyAutoAdd.ts

```diff
@@ -82,81 +86,82 @@ export class KeyAutoAddService {
     return this.tab?.id;
   }
 
-  private async process(url: string, payload: BeginPayload, tabId?: TabId) {
+  private async process(
+    url: string,
+    payload: BeginPayload,
+    existingTabId?: TabId,
+  ): Promise<unknown> {
     const { resolve, reject, promise } = withResolvers();
+    try {
+      const tab = await reuseOrCreateTab(this.browser, url, existingTabId);
```

Comment:
Let it throw. It's unexpected and should not happen. No need a try-catch for `reuseOrCreateTab`.